code/__defines/traits.dm

```diff
@@ -0,0 +1,64 @@
+#define TRAIT_LEVEL_EXISTS   0
+#define TRAIT_LEVEL_MINOR    1
+#define TRAIT_LEVEL_MODERATE 2
+#define TRAIT_LEVEL_MAJOR    3
+
+#define TRAITS_PHYSICAL  BITFLAG(0)
```

Comment:
is this define unused? downstream we use `aspect_flags` for stuff iirc, but it seems that's gone and now we just have `reapply_on_rejuvenation`? those two hardly feel equivalent

Reply:
i kind of feel that physical/mental traits are an important distinction outside of just reapplying on rejuv though, but I guess i don't have anything concrete. downstream we stored physical aspects on a 'body record' file and mental ones on a 'mind backup' file, but i guess i could just readd that distinction downstream. mostly i just see three categories:
- physical traits that initialize something about your character, like prostheses. if your character were cloned or copied, these would be lost
- physical traits that represent something constitutive about your character, like allergies. if your character were cloned or copied, these would carry over
- mental traits, which are a bit finnicky because you start getting into fuzzy mind/brain stuff like "do you keep mental aspect X if your mind is uploaded into a computer, or is it part of your Brain?"
etc.
i dunno, i guess that'd be a bridge to cross later when we have stuff that actually uses it. are aspects stored in mob snapshots at all?